src/Sql/Sql/ManagedInstanceOperation/Cmdlet/CancelAzureSqlManagedInstanceOperation.cs

```diff
@@ -0,0 +1,170 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+using System.IO;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.ManagedInstanceOperation.Model;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedInstanceOperation.Cmdlet
+{
+    /// <summary>
+    /// Defines the Get-AzSqlInstance cmdlet
+    /// </summary>
+    [Cmdlet("Stop", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceOperation",
+        DefaultParameterSetName = StopByNameAndManagedInstanceAndResourceGroupParameterSet,
+        SupportsShouldProcess = true),
+        OutputType(typeof(AzureSqlManagedInstanceOperationModel))]
+    public class RemoveAzureSqlManagedInstanceOperation : ManagedInstanceOperationCmdletBase
```

Comment:
The cmdlet is called `stop-` and class name is `Remove` while file name is `Cancel`. We might as well make them all be `Stop` so it won't be confusing